### DISCUSSION TITLE
Welcome to jackson-future-ideas Discussions!

### DISCUSSION THREAD

**Discussion by cowtowncoder** [FasterXML/jackson-future-ideas#71] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community.

  This particular repo is -- as per its README -- currently (Dec 2024) focused on making Jackson 3.0.0 reality. So all Discussions are to be related to that goal, or possible future expansions or Jackson 3.x.

**Comment by cowtowncoder** [comment]
Update: with 3.0.0 GA out, Discussions can relate to any and all future Jackson things: most likely new JSTEPs.

**Reply by JooHyukKim** [reply]
Nice, anything else I could be preparing for?

**Reply by cowtowncoder** [reply]
Was thinking of proposing changing default branches for all repos from `2.x` to `3.x`, could start a Discussion?
(not worth writing JSTEP for tho).

And could probably decide on/complete JSTEP for Long-Term Support (https://github.com/FasterXML/jackson-future-ideas/wiki/JSTEP-13).
